### PR TITLE
feat(system-backup/restore): chart, YAML update

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/1455

### CODE REVIEW

File: chart/templates/deployment-webhook.yaml [hunk 1]
```diff
@@ -164,3 +167,7 @@ spec:
         {{- end }}
       {{- end }}
       serviceAccountName: longhorn-service-account
+      volumes:
+      - name: longhorn
+        hostPath:
+          path: /var/lib/longhorn/
```
Comment (innobead) [inline chart/templates/deployment-webhook.yaml:173]: Why need this?

**Review by innobead — CHANGES_REQUESTED** [review]

File: chart/templates/deployment-webhook.yaml (hunk 1 above)
Comment (innobead) [reply chart/templates/deployment-webhook.yaml:173]: I remembered we have a privileged issue related to webhooks, so we don't want to do some validation but rely on the host. @derekbit do you remember that?

Comment (c3y1huang) [reply chart/templates/deployment-webhook.yaml:173]: We had no validation against the backup store.

Comment (derekbit) [reply chart/templates/deployment-webhook.yaml:173]: > The validator will execute the engine binary to check if the backup already exists in the backup store.

It's a newly added feature, right?

> I remembered we have a privileged issue related to webhooks, so we don't want to do some validation but rely on the host. @derekbit do you remember that?

Yes, we limit the permission
```
        securityContext:
          runAsUser: 2000
```

Comment (c3y1huang) [reply chart/templates/deployment-webhook.yaml:173]: https://github.com/longhorn/longhorn/issues/3701

Comment (c3y1huang) [reply chart/templates/deployment-webhook.yaml:173]: Reverted the privilege change.

Comment (c3y1huang) [reply chart/templates/deployment-webhook.yaml:173]: https://github.com/longhorn/longhorn-manager/pull/1593

**Review by innobead — APPROVED** [review]

**Pull request merged**